adaezeodurukwe/iReport
# #162468337 Set up database

**What does this PR do?**
Sets up project backend with PostgreSQL

**Description of work done**
- Download pg
- Set up the connection to the database
- Create the table schema

**How can this be tested?**
- Clone the repo and CD into it
-  Checkout to ch-database-setup-162468337 branch and run `npm run devstart`
NB: you should have postgres db set up and connection string configured accordingly

**Outcome**
- Users and records tables created

**What are the relevant pivotal tracker stories?**
#162468337
